### PR TITLE
[更新完毕] global settings

### PR DESCRIPTION
**Pull request by isdanni**
#45 

Btw, I find it might be easier if some of us could have write access to this repository, so review & update stage would be faster? :thinking:

### CODE REVIEW

**Comment by archibate** [comment]
Yes! Having more than one people with write access is really a good idea. I'm recently more occupied in Taichi main repo, and given that @isdanni and @ArkhamWJZ has done many significant contribution to this repo, @yuanming-hu again, could you grant them with write-access?

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 1]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
+"为CUDA指定预分配的内存大小：``export TI_DEVICE_MEMORY_GB=0.5`` 或 ``ti."
+"init(device_memory_GB=0.5)``。"
 
 #: ../../global_settings.rst:10
 msgid ""
-"To disable unified memory usage on CUDA: ``export "
-"TI_USE_UNIFIED_MEMORY=0``."
-msgstr ""
+"To disable unified memory usage on CUDA: ``export TI_USE_UNIFIED_MEMORY=0``."
+msgstr "在CUDA上禁用统一内存使用： ``导出TI_USE_UNIFIED_MEMORY = 0``。"
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "禁止 CUDA 使用统一内存： ``export TI_USE_UNIFIED_MEMORY = 0``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 2]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
 
 #: ../../global_settings.rst:8
-#, fuzzy
 msgid ""
-"To specify which GPU to use for CUDA: ``export "
-"CUDA_VISIBLE_DEVICES=[gpuid]``."
-msgstr "要指定使用哪个Arch： ``export TI_ARCH=cuda``"
+"To specify which GPU to use for CUDA: ``export CUDA_VISIBLE_DEVICES=[gpuid]``."
+msgstr ""
+"指定用于CUDA的GPU，请执行以下操作：``export CUDA_VISIBLE_DEVICES=[gpuid]``。"
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
"指定 CUDA 所使用的 GPU，请执行以下操作：``export CUDA_VISIBLE_DEVICES=[gpuid]``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 3]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
 
 #: ../../global_settings.rst:22
-#, fuzzy
 msgid ""
-"Restart the entire Taichi system (destroy all tensors and kernels): "
-"``ti.reset()``."
-msgstr "重新启动Taichi运行系统（清除内存，销毁所有变量和内核）： ``ti.reset()``"
+"Restart the entire Taichi system (destroy all tensors and kernels): ``ti."
+"reset()``."
+msgstr "重新启动Taichi运行系统（销毁所有量和内核）： ``ti.reset()``。"
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "重新启动 Taichi 运行系统（销毁所有张量和内核）： ``ti.reset()``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 4]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
+msgstr "日志记录"
 
 #: ../../global_settings.rst:28
 msgid ""
-"Show more detailed log to level TRACE: ``export TI_LOG_LEVEL=trace`` or "
-"``ti.set_logging_level(ti.TRACE)``."
+"Show more detailed log to level TRACE: ``export TI_LOG_LEVEL=trace`` or ``ti."
+"set_logging_level(ti.TRACE)``."
 msgstr ""
+"显示更详细的日志到TRACE级别：``export TI_LOG_LEVEL=trace`` 或 ``ti."
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
"通过输出 TRACE 级日志以显示更详尽的信息：``export TI_LOG_LEVEL=trace`` 或 ``ti."
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 5]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
+"set_logging_level(ti.TRACE)``。"
 
 #: ../../global_settings.rst:29
-#, fuzzy
 msgid ""
 "Eliminate verbose outputs: ``ti.get_runtime().set_verbose(False)`` or "
 "``TI_VERBOSE=0``."
-msgstr "消除详细输出：  ``ti.get_runtime().set_verbose(False)``"
+msgstr ""
+"消除详细输出： ``ti.get_runtime().set_verbose(False)`` 或 ``TI_VERBOSE=0``。"
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
"消除冗余输出： ``ti.get_runtime().set_verbose(False)`` 或 ``TI_VERBOSE=0``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 6]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
+"TI_CACHE_RUNTIME_BITCODE=1``."
 msgstr ""
+"以 **开发模式** 缓存编译后的运行时位码以节省启动时间：``export "
+"TI_CACHE_RUNTIME_BITCODE=1``。"
 
 #: ../../global_settings.rst:35
-#, fuzzy
 msgid "To trigger GDB when Taichi crashes: ``export TI_GDB_TRIGGER=1``."
-msgstr "崩溃时不触发GDB：``export TI_GDB_TRIGGER=0``"
+msgstr "崩溃时不触发GDB：``export TI_GDB_TRIGGER=1``。"
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
msgstr "当 Taichi 崩溃时触发 GDB：``export TI_GDB_TRIGGER=1``。"
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 7]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
+"init(print_ir=True)``。"
 
 #: ../../global_settings.rst:17
 msgid ""
-"To print preprocessed Python code: ``export TI_PRINT_PREPROCESSED=1`` or "
-"``ti.init(print_preprocessed=True)``.."
+"To print preprocessed Python code: ``export TI_PRINT_PREPROCESSED=1`` or ``ti."
+"init(print_preprocessed=True)``.."
 msgstr ""
+"要打印预处理的Python代码：``export TI_PRINT_PREPROCESSED=1`` 或 ``ti."
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
"打印预处理后的 Python 代码：``export TI_PRINT_PREPROCESSED=1`` 或 ``ti."
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 8]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
+"指定用于CUDA的GPU，请执行以下操作：``export CUDA_VISIBLE_DEVICES=[gpuid]``。"
 
 #: ../../global_settings.rst:9
-#, fuzzy
 msgid ""
 "To specify pre-allocated memory size for CUDA: ``export "
 "TI_DEVICE_MEMORY_GB=0.5`` or ``ti.init(device_memory_GB=0.5)``."
-msgstr "要为CUDA指定预分配的内存大小： ``export TI_DEVICE_MEMORY_GB=0.5``"
+msgstr ""
+"为CUDA指定预分配的内存大小：``export TI_DEVICE_MEMORY_GB=0.5`` 或 ``ti."
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
"指定 CUDA 预分配内存的大小：``export TI_DEVICE_MEMORY_GB=0.5`` 或 ``ti."
```

File: locales/zh_CN/LC_MESSAGES/global_settings.po [hunk 9]
```diff
@@ -8,124 +8,132 @@ msgstr ""
...
+msgstr "开发"
 
 #: ../../global_settings.rst:34
 msgid ""
-"Cache compiled runtime bitcode in **dev mode** to save start up time: "
-"``export TI_CACHE_RUNTIME_BITCODE=1``."
+"Cache compiled runtime bitcode in **dev mode** to save start up time: ``export "
+"TI_CACHE_RUNTIME_BITCODE=1``."
 msgstr ""
+"以 **开发模式** 缓存编译后的运行时位码以节省启动时间：``export "
```
Comment (ArkhamWJZ) [inline locales/zh_CN/LC_MESSAGES/global_settings.po]:
```suggestion
"以 **开发模式** 缓存编译后运行时位码(compiled runtime bitcode)以节省启动时间：``export "
```

**Review by ArkhamWJZ — COMMENTED** [review]
LGTM🔥! + only a few changes that need to be made, I think 🙆‍♂️

**Review by archibate — APPROVED** [review]
Cool!

**Pull request merged**